lib/ups/builders/builder_base.rb

```diff
@@ -144,7 +144,7 @@ def add_ship_from(opts = {})
       # @return [void]
       def add_package(opts = {})
         shipment_root << Element.new('Package').tap do |org|
-          org << packaging_type
+          org << packaging_type(opts[:packaging_type])
```

Comment:
Is it okay to send both packaging type and dimensions? Also what if `opts[:packaging_type]` is not passed, we will still try to build the packaging type here